### DISCUSSION TITLE
Show for some players

### DISCUSSION THREAD

**Discussion by Ulvtand** [mclemente/fvtt-module-polyglot#197] [Q&A] [answered]
Can I somehow set up the players who know the language to get a translation while others do not? 
This applies to the Witchwe TRPG.

**Comment by mclemente** [comment, marked as answer]
Create any item named with this pattern `Language (Name)` and it should show up. I suggest Notes, in the Background section, since they're out of the way.
![image](https://user-images.githubusercontent.com/5288872/138156791-3a1e06af-c4b5-4857-95d6-18f0eb10644f.png)

**Reply by Ulvtand** [reply]
I do not know where to do it? Unfortunately, I do not follow the description

**Reply by mclemente** [reply]
In the actor sheet, on the Background tab, at the end there's a `Notes +` section, you add one and name it like in the image.

**Reply by Ulvtand** [reply]
Yep, I found it.

**Reply by Ulvtand** [reply]
But since then? How do I use it to make the text readable?

**Reply by Ulvtand** [reply]
I feel unusually stupid ... ;)

**Reply by mclemente** [reply]
Add languages into the Custom Languages setting, they will show up on the language selector, then players with the actual languages will be able to read them.

**Reply by Ulvtand** [reply]
TOP, thank you so much for the help <3
